### ISSUE TITLE
ModuleNotFoundError: No module named 'html.parser'

### ISSUE THREAD

**Issue by zersiax** [yplassiard/nvda-translate#7]
Looks like this addon currently doesn't work with the latest NVDA snapshots, as this error is returned. Full log:
ERROR - globalPluginHandler.listPlugins (14:37:07.393) - MainThread (37948):
Error importing global plugin translate
Traceback (most recent call last):
  File "C:\Users\flori\AppData\Roaming\nvda\addons\translate\globalPlugins\translate\__init__.py", line 33, in <module>
    import mtranslate
  File "C:\Users\flori\AppData\Roaming\nvda\addons\translate\globalPlugins\translate\mtranslate\__init__.py", line 2, in <module>
    from .core import translate
  File "C:\Users\flori\AppData\Roaming\nvda\addons\translate\globalPlugins\translate\mtranslate\core.py", line 36, in <module>
    import html.parser
ModuleNotFoundError: No module named 'html.parser'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "globalPluginHandler.pyc", line 23, in listPlugins
  File "importlib\__init__.pyc", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\flori\AppData\Roaming\nvda\addons\translate\globalPlugins\translate\__init__.py", line 37, in <module>
    logHandler.log.exception("Failed to initialize translate addon", e)
  File "logHandler.pyc", line 192, in exception
TypeError: 'ModuleNotFoundError' object is not subscriptable

It'd be awesome if this could be fixed :)

**Comment by zersiax** [comment]
*bump* This problem will make the addon not work with NVDA 2021.1 which will probably enter beta soon. It'd be great if this could be looked into :)

**Comment by yplassiard** [comment]
Thanks, will look at it shortly.

**Comment by yplassiard** [comment]
This has been fixed in the latest TRanslate 2021.06.1 page.
